src/WebJobs.Script.WebHost/Configuration/CorsOptionsSetup.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Cors.Infrastructure;
+using Microsoft.Extensions.Options;
+
+namespace Microsoft.Azure.WebJobs.Script.WebHost.Configuration
+{
+    public class CorsOptionsSetup : IConfigureOptions<CorsOptions>
+    {
+        private readonly IEnvironment _env;
+        private readonly IOptions<HostCorsOptions> _hostCorsOptions;
+
+        public CorsOptionsSetup(IEnvironment env, IOptions<HostCorsOptions> hostCorsOptions)
+        {
+            _env = env;
+            _hostCorsOptions = hostCorsOptions;
+        }
+
+        public void Configure(CorsOptions options)
+        {
+            if (true && _hostCorsOptions.Value != null)
```

Comment:
Why true && ?

Reply:
Whoops. That was for local testing; needs to be switched back. Thanks for the catch.